seqr/utils/file_utils.py

```diff
@@ -32,8 +45,23 @@ def does_file_exist(file_path):
 
 def file_iter(file_path, byte_range=None, raw_content=False):
     if _is_google_bucket_file_path(file_path):
-        for line in _google_bucket_file_iter(file_path, byte_range=byte_range, raw_content=raw_content):
-            yield line
+        path_segments = file_path.split('/')
+        if len(path_segments) < 4:
```

Comment:
Why 4 path segments?

Reply:
It's because of the double slash after `gs:`:
1. `'gs:'`
2. `''` (empty)
3. `<bucket_name>`
4. `<path>`